modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -38,24 +38,6 @@ definition {
 
 	}
 
-	@description = "Verify that any user with administrator role can reassign allowed tasks"
-	@ignore = "Test Stub"
-	@priority = 5
-	test AdministratorUserCanReassignAnyTasks {
-
-		// TODO AdministratorUserCanReassignAnyTasks pending implementation
-
-	}
-
-	@description = "Verify that an administrator user that it's not asset creator can reassign multiple already assigned tasks to different users"
```

Comment:
Removed as it is already covered in another test https://liferay.atlassian.net/browse/LRQA-74799